assets/src/styles/blocks/Spreadsheet.scss

```diff
@@ -28,10 +28,18 @@
   width: 350px;
   max-width: 100%;
   margin: 10px 0;
-  padding: 3px 8px;
+  padding: 0 8px;
+  height: 40px;
   border: 1px solid;
 }
 
+.spreadsheet-default-sort {
+  height: 40px !important;
```

Comment:
Not ideal, but this will be removed after a tiny refactor documented in this ticket: https://jira.greenpeace.org/browse/PLANET-4740

Reply:
Will this work on all screen sizes?

Reply:
Hmmm I'll double check, this should be the same height as the input

Reply:
Did you verify that this works?

Reply:
Yes, I actually did a few changes to make it look good on all sizes.